app.py

```diff
@@ -55,6 +59,31 @@
 def main_page():
     return render_template('index.html')
 
+@auth.verify_password
+def verify_password(user_id, password):
+    return user_id
+
+@app.route('/forms', methods=['POST'])
+@auth.login_required
+def create_form():
+    form_id = random.randint(0, 2**32 - 1)
```

Comment:
You can already use a uuid.

Reply:
This also does the trick until the datastore gives out actual ids for the forms :) 
```python
import uuid

id=uuid.uuid4()
```